### ISSUE TITLE
Syncing with Terrain Layer Module to apply difficult terrain

### ISSUE THREAD

**Issue by Avitale34** [wsaunders1014/ShowDragDistance#41]
The new terrain module allows you to put multipliers down on the map for difficult terrain is it possible to have showdragdistance talk with those points? Not an issue so you can remove if you'd like I just wanted to suggest, love your module!

**Comment by wsaunders1014** [comment]
Yep, i built Terrain Layer in mind with ShowDragDistance, I plan on integrating it soon.
